### DISCUSSION TITLE
PIPELINE_LOAD_ERROR: An HTTP status code indicating failure was received: 403

### DISCUSSION THREAD

**Discussion by Gamedirection** [ViewTube/viewtube#2967] [Q&A]
After watching 30 seconds on any video it seems to hit this error.
If you need more information please let me know.
Hosting using Portainer --> https://github.com/ViewTube/viewtube/blob/development/docker-compose.yml

**Comment by Gamedirection** [comment]
An error occurs 33 seconds into playback. I attempted to adjust the Ad Block settings both on browser and in app and tested both HTTP and HTTPS configurations. The issue consistently arises within the first 30 to 40 seconds of playback. 
I tried - 3 videos with various copywrite checks. (generic Minecraft video, a popular music video, and a recommended video)

Is this a known issue with YouTube, or could I be making an error in the setup?

**Comment by unknown-untitled** [comment]
same
